lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +179,37 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.urls = []
+        self._config = config
+        self._seed_urls = config.get_seed_urls()
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        url = article_bs['href']
+        if (isinstance(url, str)) and (url.count('/') > 3) and (url.startswith('/news/')):
+            return url
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
-        pass
+        for url1 in self._seed_urls:
+            response = make_request(url1, self._config)
+            if response.status_code == 200:
+                main_bs = BeautifulSoup(response.text, 'lxml')
+                for url2 in main_bs.find_all('a'):
+                    url3 = self._extract_url(url2)
```

Comment:
consider naming variables in a more comprehensible way than `url1`, `url2`, `url3`